eventlogger/src/main/java/gov/cms/ab2d/eventlogger/LoggableEvent.java

```diff
@@ -0,0 +1,10 @@
+package gov.cms.ab2d.eventlogger;
+
+import java.time.OffsetDateTime;
+
+/**
+ * Interface describing a loggable event
+ */
+public interface LoggableEvent {
+    boolean log(OffsetDateTime eventTime);
```

Comment:
What does return value (true/false) indicate?

Reply:
That it was able to successfully log the event. Not sure if we'll ever complain about it. What are we going to do, log it?